File: FlowCrypt/src/main/java/com/flowcrypt/email/util/exception/NodeEncryptException.java
/*
 * © 2016-2019 FlowCrypt Limited. Limitations apply. Contact human@flowcrypt.com
 * Contributors: DenBond7
 */

package com.flowcrypt.email.util.exception;

/**
 * This exception can occur during the encryption process.
 *
 * @author Denis Bondarenko
 * Date: 1/25/19
 * Time: 6:34 PM
 * E-mail: DenBond7@gmail.com
 */
public class NodeEncryptException extends NodeException {
  public NodeEncryptException(String message) {
    super(message);
  }
}

File: FlowCrypt/src/main/java/com/flowcrypt/email/util/exception/NodeEncryptException.kt
/*
 * © 2016-present FlowCrypt a.s. Limitations apply. Contact human@flowcrypt.com
 * Contributors: DenBond7
 */

package com.flowcrypt.email.util.exception

import com.flowcrypt.email.api.retrofit.response.base.ApiError

/**
 * This exception can occur during the encryption process.
 *
 * @author Denis Bondarenko
 * Date: 1/25/19
 * Time: 6:34 PM
 * E-mail: DenBond7@gmail.com
 */
class NodeEncryptException(error: ApiError) : NodeException(error)
